pybossa/view/projects.py

```diff
@@ -949,6 +949,20 @@ def handle_valid_form(form):
                     prodsubprods=prodsubprods)
     return handle_content_type(response)
 
+
+@blueprint.route('/<short_name>/remove-password', methods=['POST'])
+@login_required
+@csrf.exempt
+def remove_password(short_name):
+    if current_app.config.get('PROJECT_PASSWORD_REQUIRED'):
```

Comment:
The "remove password" link can also be hidden in the UI if this value is set in the config. See [here](https://github.com/bloomberg/pybossa-default-theme/pull/459/files#diff-39ee427448a2ae0a8931fd85fb0d77eccbbe4f22609a7af1d7533f7a8922a88eR66) and add a conditional similar to [this](https://github.com/bloomberg/pybossa-default-theme/blob/main/templates/projects/project.html#L67C21-L67C58).

```html
{% if config.PROJECT_PASSWORD_REQUIRED %}
  <form id="remove-password-form" method="post" action="{{ url_for('project.remove_password', short_name = project.short_name) }}">
      <div class="form-actions">
          <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
          <input type="hidden" name="password" value="" />
          <button type="submit" class="btn btn-link" value="Remove Project Password">{{_('Remove Project Password')}}</button>
      </div>
  </form>
{% endif %}
```

Reply:
implemented via b613336